crates/rooch-types/src/chain_id.rs

```diff
@@ -178,6 +198,11 @@ impl BuiltinChainID {
     pub fn genesis_ctx(&self) -> GenesisContext {
         let chain_id = self.chain_id().id;
         match self {
+            BuiltinChainID::Local => {
+                //Local timestamp from 0, developer can manually set the timestamp
```

Comment:
Shall it be fetched in local timestamp in UTC?

Reply:
Using local time will cause an undetermined genesis package.